wercker.yml

```diff
@@ -4,17 +4,45 @@ build:
 
   steps:
 
-    - setup-go-workspace
-
+    - setup-go-workspace:
+        package-dir: github.com/yieldr/terraform-provider-auth0
+    
     - script:
         name: build
         code: |
           make build
 
+    - script:
+        name: package
+        code: |
+          make clean packages BUILD_PATH=$WERCKER_OUTPUT_DIR
+
     - script:
         name: run tests
         code: |
           make testacc OPTS=-coverprofile=c.out
 
     - yieldr/test-reporter:
         id: $CODECLIMATE_REPO_TOKEN
+    
+deploy:
+
+  steps:
+
+    - github-create-release:
+        token: $GITHUB_TOKEN
+        tag: v1.0.0
```

Comment:
Not sure if you had a release strategy in mind already but this is the quickest way I knew how to get this rolling. I don't have much experience with wercker so I just threw this together. If you're familiar with semantic-release, we could also opt for https://github.com/io-monad/wercker-step-semantic-release .